api/src/controllers/login.js

```diff
@@ -51,6 +62,31 @@ const templates = {
       'privacy.policy'
     ],
   },
+  passwordReset: {
+    file: path.join(__dirname, '..', 'templates', 'login', 'password-reset.html'),
+    translationStrings: [
+      'login.show_password',
+      'login.hide_password',
+      'change.password.title',
+      'change.password.hint',
+      'change.password.submit',
+      'change.password.new.password',
+      'change.password.confirm.password',
+      'password.weak',
+      'password.length.minimum',
+      'password.must.match',
+      'user.password.current',
+      'password.current.incorrect',
+      'password.same'
+    ],
+  }
+};
+
+const skipPasswordChange = async (user, userCtx) => {
+  return !user?.password_change_required ||
+    user?.token_login?.active ||
+    roles.isDbAdmin(userCtx) ||
```

Comment:
Yes, you're correct. It _might_  be extra validation since those checks are already in `user.js`. Maybe lets keep it because after the after the POST in login which needs to decide where to go next, these checks make it clear in this controller.